cms/djangoapps/contentstore/views/certificates.py

```diff
@@ -322,9 +322,6 @@ def certificate_activation_handler(request, course_key_string):
     POST
         json: is_active. update the activation state of certificate
     """
-    # Only global staff (PMs) are able to activate/deactivate certificate configuration
```

Comment:
@mattdrayer  - The proposal is that it inherits from course access rules that dictate ability to view this page (which I think is somewhere above this location in certificates.py. Does that make sense? Any other way I could think about it?